frontend/website/pages/docs/tokens.mdx

```diff
@@ -0,0 +1,112 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Tokens"});
+
+# Tokens  
+
+Using the Coda CLI (command-line interface) is the primary way to create new token accounts, create new tokens, and mint tokens on the Coda blockchain. 
+
+In this section, you will find out how to use the Coda CLI to create new tokens, create new token accounts, and mint non-default tokens!
+
+## Creating Tokens
+
+With the CLI, you are able to create your own layer 2 tokens! Layer 2 tokens are a new type of token that is introduced to the Coda blockchain. These new types of tokens have a unique identifier that differentiate them from all other tokens in the blockchain. Users with an associated token account can then receive/send this new type of token that is totally separate from the native Coda token itself!  When you create a new token, you are then able to mint more of the token to be used (see below in Minting Tokens). 
+
+When creating a new token, a new token ID is created as well as a token account public key. This token account owns the token and can mint more of the token. The account will also (eventually) have the power to enable/disable accounts, and to set whether new accounts may be created for the token.
+
+
+There is a fee to create a new token. The fee is the amount you are willing to pay to process the transaction. If this flag is not specified, the default value is 5 Coda. Additionally, the fee for creating the new token account is charged in addition to the transaction fee. 
+
+
+### Create a new token
+
+```
+$ coda client create-token -sender <PUBLIC_KEY>
+```
+
+<Alert>
+
+The required field for creating a new token account is: 
+
+- sender : The public key from which you want to send the transaction
+
+</Alert>
+
+See 
+```
+$ coda client create-token -help
+```
+ 
+to learn more about how to use the command.
+
+
+
+## Creating Token Accounts
+
+With the CLI, you are able to create a token account for existing tokens. Token accounts are like regular accounts but solely meant to interact with the layer 2 tokens that exist in the Coda protocol. There is a unique token account for each type of token you wish to interact with. 
```

Comment:
> There is a unique token account for each type of token you wish to interact with.

You must create a separate unique token account for each type of token you wish to interact with